src/pages/AddServers.js

```diff
@@ -168,29 +178,41 @@ class AddServers extends BaseUpdateWizardPage {
     // check if newly added servers have addresses conflicts with any deployed servers
     for (let i = 0; i < newServers.length; i++) {
       let newServer = newServers[i];
-      hasConflicts = hasConflictAddresses(newServer, modelDeployedServers);
-      if (hasConflicts) {
-        break;
+      isServersInvalid = hasConflictAddresses(newServer, modelDeployedServers);
```

Comment:
No, you still don't need the variable or any of the checks of its value. This code, and all of the other checks below, should look something like:
```
if (hasConflictAddresses(newServer, modelDeployedServers))
  return true;
```